### PR TITLE
no self.callPackge

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

File: ocaml/default.nix [hunk 1]
```diff
@@ -23,6 +23,7 @@
 , cairo
 , gtk2
 , zlib-oc
+, callPackage
```
Comment (ulrikstrid) [inline ocaml/default.nix:26]: Should come from osuper?

Comment (anmonteiro) [reply ocaml/default.nix:26]: no, it doesn't even exist.. It comes from `pkgs.callPackage`.

Comment (ulrikstrid) [reply ocaml/default.nix:26]: It exists and scopes you with the correct OCaml version

Comment (anmonteiro) [reply ocaml/default.nix:26]:
```
          468|   dune_3 = lib.trace "${builtins.toJSON (osuper.callPackage)}" osuper.dune_3.overrideAttrs (o: {
```

**Comment by anmonteiro** [comment]
maybe it was oself after all

**Pull request closed without merging**